### PR TITLE
Sidebar fix, store modifications, FollowerComponent minor upgrade

### PR DESCRIPTION
**Pull request by laszloforgacs**

### CODE REVIEW

File: minitwit_admin/src/compositionStore/index.js [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
+    initStore,
+    useSidebar,
+    useAsync,
+    useTwits,
+    useUsers,
+    useFollowers,
+    async,
+    twits,
+    users,
+    followers
```
Comment (hvsio) [inline minitwit_admin/src/compositionStore/index.js:33]: I don't think we need to export anything from index.js. The provide/inject is supposed to ease passing components from parent to distant children so I guess if we are calling initStore() on App.vue level, each other vue component embedded in our app will be able to retrieve the provided value on its own level (fx. UserForm in setup() can do inject("useUsers") to get access to the methods).

TLDR: I think its a bit contraintuitive to provide and inject on the same level (ref: https://vuejs.org/guide/components/provide-inject.html#inject).

**Review by hvsio — CHANGES_REQUESTED** [review]

File: minitwit_admin/src/compositionStore/index.js (hunk 1 above)
Comment (laszloforgacs) [reply minitwit_admin/src/compositionStore/index.js:33]: Yes, they are there because I did not want to break existing functionality, there are components still using them.
And by importing useSomething() you dont have to use inject("useSomething") in the setup method, I think its more convenient that way. UserProfileScreen.vue is a good example

Comment (hvsio) [reply minitwit_admin/src/compositionStore/index.js:33]: alrighty then, just making sure its a conscious choice

**Review by hvsio — APPROVED** [review]
